x/distribution/client/cli/tx.go

```diff
@@ -52,13 +52,12 @@ func newSplitAndApply(
 	genOrBroadcastFn newGenerateOrBroadcastFunc, clientCtx client.Context,
 	fs *pflag.FlagSet, msgs []sdk.Msg, chunkSize int,
 ) error {
-
-	if chunkSize == 0 {
+	totalMessages := len(msgs)
+	if chunkSize == 0 || totalMessages == 0 {
 		return genOrBroadcastFn(clientCtx, fs, msgs...)
```

Comment:
Why would be broadcast a tx with 0 msgs if it's going to be rejected by the node? Shouldn't we just throw an error early?